main.nf

```diff
@@ -24,72 +24,94 @@ def helpMessage() {
+      --save_nonrRNA_reads          Save FastQ file intermediates after removing rRNA
+      --rRNA_database_manifest      Path to file that contains file paths for rRNA databases, optional
+
+    Alignment:
+      --aligner                     Specifies the aligner to use (available are: 'hisat2', 'star')
+      --pseudo_aligner              Specifies the pseudo aligner to use (available are: 'salmon'). Runs in addition to `--aligner`
+      --stringTieIgnoreGTF          Perform reference-guided de novo assembly of transcripts using StringTie i.e. dont restrict to those in GTF file
+      --seq_center                  Add sequencing center in @RG line of output BAM header
+      --saveAlignedIntermediates    Save the BAM files from the aligment step - not done by default
+      --saveUnaligned               Save unaligned reads from either STAR, HISAT2 or Salmon to extra output files.
+      --skipAlignment               Skip alignment altogether (usually in favor of pseudoalignment)
+
+    Read Counting:
+      --fc_extra_attributes         Define which extra parameters should also be included in featureCounts (default: 'gene_name')
+      --fc_group_features           Define the attribute type used to group features. (default: 'gene_id')
+      --fc_count_type               Define the type used to assign reads. (default: 'exon')
+      --fc_group_features_type      Define the type attribute used to group features based on the group attribute (default: 'gene_biotype')
+
+    QC:
+      --skipQC                      Skip all QC steps apart from MultiQC
+      --skipFastQC                  Skip FastQC
+      --skipPreseq                  Skip Preseq
+      --skipDupRadar                Skip dupRadar (and Picard MarkDuplicates)
+      --skipQualimap                Skip Qualimap
+      --skipBiotypeQC               Skip Biotype QC
+      --skipRseQC                   Skip RSeQC
+      --skipEdgeR                   Skip edgeR MDS plot and heatmap
+      --skipMultiQC                 Skip MultiQC
+
+    Other options
+      --sampleLevel                 Used to turn off the edgeR MDS and heatmap. Set automatically when running on fewer than 3 samples
       --outdir                      The output directory where the results will be saved
       -w/--work-dir                 The temporary directory where intermediate data will be saved
       --email                       Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      --sampleLevel                 Used to turn of the edgeR MDS and heatmap. Set automatically when running on fewer than 3 samples
+      --email_on_fail               Same as --email, except only send mail if the workflow is not successful
       --maxMultiqcEmailFileSize     Theshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
```

Comment:
```suggestion
      --max_multiqc_email_size     Theshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
```